core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -50,6 +50,9 @@ object ConsumerPerformance {
     val totalBytesRead = new AtomicLong(0)
     val consumerTimeout = new AtomicBoolean(false)
 
+    if (config.useOldConsumer)
+      Console.err.println("Old consumer is deprecated and will be removed in a future release. Consider using the new consumer.")
```

Comment:
We should not deprecate this as it's useful to compare the performance of the old and new consumers.